modules/eventing/pages/eventing-lifecycle.adoc

```diff
@@ -0,0 +1,138 @@
+This logging attribute is function-specific and is not applicable for the Eventing Service in general.
+* *N1QL Consistency:* The default consistency level of N1QL statements in the handler either None (the Default) or Request.
+The default is 3 (the minimum is 1 maximum is 64).
+* *Workers:* The number of worker threads to be allocated to the Function.
+The default is 3 (the minimum is 1 maximum is 64).
+* *Language compatibility:* The language version of the handler for backward compatibility either 6.0.0 or 6.5.0. Primarily to allow older handlers to continue to see the runtime behavior that existed at the time they were authored
+* *Script Timeout:* The number of seconds that should elapse before the script times out and is terminated.
+The default is 60.
+
+| Bindings
+| A binding is a construct that allows separating environment specific variables (example: bucket names, external endpoint URLs, credentials) from the handler source code.  Currently Eventing Functions support either *Bucket Bindings* (to access KV data) or *URL Bindings* to communicate externally via cURL. An Eventing Function can have no binding, one binding, or several bindings.
+
+For more information on Bindings, refer to xref:eventing-Terminologies.adoc#section_mzd_l1p_m2b[Bindings].
+|===
+
+. *Prerequisites*: In order to show the basic life cycle of adding an Eventing Function we need a 'source' bucket and a metadata *metadata*. You must have a bucket typically called *metadata* used solely by Eventing if it is misisng create it with a minimum size of 100MB.  For the source bucket we will use 'travel-sample' sample document set. 
++
+If you don’t already have the sample bucket *beer-sample* listed in the *Couchbase Web Console* > *Buckets* page you can load this document set as follows:
+
+** Access the *Couchbase Web Console* > *Settings* page
+** Select the *Sample Buckets* in the upper right banner. 
+** Check *beer-sample* checkbox.
+** Click *Load Sample Data*.
+
+. To get to the next screen *ADD FUNCTION* dialog, as follows:
+ ** For the *Source Bucket* drop-down, select *beer-sample*.  
+ ** For the *Metadata Bucket* drop-down, select *metadata*.
+ ** Enter *my_evt_function* as the name of the Function you are creating in the *Function Name* text-box.
+ ** [Optional Step] Enter text *A simple Eventing Function only prints IDs*, in the *Description* text-box.
+ ** For the *Settings* option, use the default values.
+ ** For the *Bindings* option, don't add any bindings (we will merely be logging messages).
++
+image::addfunc_02_settings.png[,484]
+. After providing all the required information in the *ADD FUNCTION* dialog, click *Next: Add Code*.
+The *delete_orphaned_txns* dialog appears.
+** The *my_evt_function* dialog initially contains a placeholder code block.
+You will accept the default for your *my_evt_function code*.
++
+image::addfunc_03_editor_with_default.png[,100%]
+** Click *Save*.
```

Comment:
I found that the **Save** button was unavailable. I had to make at least one change and undo it before I could save the boilerplate code. Please check if this step is necessary and remove it if not.

Reply:
You are correct it comes in as a default and a Save is not neeeded.